static/openapi.json

```diff
@@ -0,0 +1,537 @@
+        "parameters": [
+          {
+            "required": true,
+            "schema": {
+              "title": "Subject"
+            },
+            "name": "subject",
+            "in": "path"
+          },
+          {
+            "required": false,
+            "schema": {
+              "title": "Details",
+              "type": "boolean",
+              "default": false
+            },
+            "name": "details",
+            "in": "query"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Successful Response",
+            "content": {
+              "application/json": {
+                "schema": {}
+              }
+            }
+          },
+          "422": {
+            "description": "Validation Error",
+            "content": {
+              "application/json": {
+                "schema": {
+                  "$ref": "#/components/schemas/HTTPValidationError"
+                }
+              }
+            }
+          }
+        }
+      }
+    },
+    "/works/{olid}": {
+      "get": {
+        "summary": "Read Works",
+        "operationId": "read_works_works__olid__get",
+        "parameters": [
+          {
+            "required": true,
+            "schema": {
+              "title": "Olid"
+            },
+            "name": "olid",
+            "in": "path"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Successful Response",
+            "content": {
+              "application/json": {
+                "schema": {}
+              }
+            }
+          },
+          "422": {
+            "description": "Validation Error",
+            "content": {
+              "application/json": {
+                "schema": {
+                  "$ref": "#/components/schemas/HTTPValidationError"
+                }
+              }
+            }
+          }
+        }
+      }
+    }
+  },
+  "components": {
+    "schemas": {
+      "HTTPValidationError": {
+        "title": "HTTPValidationError",
+        "type": "object",
+        "properties": {
+          "detail": {
+            "title": "Detail",
+            "type": "array",
+            "items": {
+              "$ref": "#/components/schemas/ValidationError"
+            }
+          }
+        }
+      },
+      "ValidationError": {
+        "title": "ValidationError",
+        "required": [
+          "loc",
+          "msg",
+          "type"
+        ],
+        "type": "object",
+        "properties": {
+          "loc": {
+            "title": "Location",
+            "type": "array",
+            "items": {
+              "type": "string"
+            }
+          },
+          "msg": {
+            "title": "Message",
+            "type": "string"
+          },
+          "type": {
+            "title": "Error Type",
+            "type": "string"
+          }
+        }
+      }
+    }
+  }
+}
```

Comment:
⛔

Reply:
Thank you so much for the review, sorry, but I didn't understand correctly what needs to be corrected here.

Reply:
Add one and only one return key to the end of the file.

Reply:
Thanks a lot, I've made the changes.